### ISSUE TITLE
Incorrect Station + System sent

### ISSUE THREAD

**Issue by Bobcat64** [Marginal/EDMarketConnector#204]
I docked at a remote system (Leunii) and was looking at EDDB for a nearby station with a shipyard. I was confused when it said there was a station called 'Li Qing Jao" that fit the bill, except it doesn't exist in-game. 

Turns out that it was me that sent data (through EDMC) saying there was a station called that in Leunii. I have most of my ships in Li Qing Jao at Sol. In fact I had the FAS I'm in shipped there last night and just now was flying around with it. 

The station, [Finlay Platform](https://eddb.io/station/23819), which I docked at doesn't have a market or many facilities. It is worth noting that the EDMC UI did display the correct station and system names, with appropriate links.

I've [uploaded the file](https://github.com/Marginal/EDMarketConnector/files/966374/Leunii.Li.Qing.Jao.2017-04-29T15.15.24.txt)  created in the menu File -> Save Raw Data while at the station, it claims the last system is Leunii and Last station is Li Qing Jao. 

[Log from ROSS](https://ross.eddb.io/eddn/log/23241137) that created the entry 
Hope this helps

**Comment by inorton** [comment]
Any chance you had a ship transfer in progress to Li Quing Jao?

**Comment by Bobcat64** [comment]
No not at the time this occurred. 
I had transferred the ship I was in last night (I didn't wait for it to complete). When I got on today I hopped into it, renamed it, swapped a fuel scoop in and went on a trek out.
Might be unrelated, but I believe in the game's map it still had a marker for where my ship WAS, but the shipyard screen confirmed it was at sol. Possible it may be an issue on FDev's side...
